doc/mundo.txt

```diff
@@ -20,11 +20,13 @@ CONTENTS                                                      *Mundo-contents*
         3.9  mundo_preview_statusline .. |mundo_preview_statusline|
              mundo_tree_statusline ..... |mundo_tree_statusline|
         3.10 mundo_auto_preview ........ |mundo_auto_preview|
-        3.11 mundo_verbose_graph ....... |mundo_verbose_graph|
-        3.12 mundo_playback_delay ...... |mundo_playback_delay|
-        3.13 mundo_mirror_graph ........ |mundo_mirror_graph|
-        3.14 mundo_inline_undo ......... |mundo_inline_undo|
-        3.15 mundo_return_on_revert .... |mundo_return_on_revert|
+        3.11 mundo_auto_preview_delay .. |mundo_auto_preview_delay|
+        3.12 mundo_verbose_graph ....... |mundo_verbose_graph|
+        3.13 mundo_playback_delay ...... |mundo_playback_delay|
+        3.14 mundo_mirror_graph ........ |mundo_mirror_graph|
+        3.15 mundo_inline_undo ......... |mundo_inline_undo|
+        3.16 mundo_inline_undo_delay ... |mundo_inline_undo_delay|
+        3.17 mundo_return_on_revert .... |mundo_return_on_revert|
```

Comment:
I can't think of a reason a user would want to use two different delay values - how about just having one `mundo_render_delay` (or whatever) that is used for both options?

Reply:
There's just the one delay value, the 'mundo_inline_undo_delay' option determines whether inline diff generation is also delayed. The name is misleading however, I'll change it to something a little more clearer, maybe 'mundo_enable_inline_delay'.